server/test/unit/server/db/test_model.py

```diff
@@ -935,6 +944,225 @@ def test_pre_delete(self, mock_lazy):
     def test_pre_delete_connect(self):
         self.assertTrue(model.signals.pre_delete.has_receivers_for(model.Importer))
 
+    def test_delete_with_certs(self):
+        """
+        Ensure that the delete() method cleans up the Importer's local storage.
+        """
+        config = {
+            importer_constants.KEY_SSL_CA_CERT: 'CA Cert',
+            importer_constants.KEY_SSL_CLIENT_CERT: 'Client Cert',
+            importer_constants.KEY_SSL_CLIENT_KEY: 'Client Key'}
+        importer = model.Importer(repo_id='coolcars', importer_type_id='supercar', config=config)
+        temp_path = tempfile.mkdtemp()
```

Comment:
Somebody told me about https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory recently. Too bad we can't use Python 3.